### ISSUE TITLE
[a11y] aria label progressbar compliance

### ISSUE THREAD

**Issue by Etat-Plateforme** [angular-ui/bootstrap#4247]
Each progressbar should be labelled and a reference to this label should be provided using aria-labelledby.

**Comment by wesleycho** [comment]
I took a look here, and it looks like Bootstrap does not use this, as can be seen [here](http://getbootstrap.com/components/#progress) - is this also a deficiency in Bootstrap?

**Comment by wesleycho** [comment]
This is addressed by #4350.
